prescriptor_robojudge.ipynb

```diff
@@ -99,15 +133,30 @@
    "outputs": [],
    "source": [
     "# Generate some random prescriptions\n",
```

Comment:
Only generate 2 random prescriptors instead of 3, since we are adding blind_greedy